code_pipeline/validation.py

```diff
@@ -65,7 +65,7 @@ def is_not_self_intersecting(self, the_test):
         return road_polygon.is_valid()
 
     def is_too_sharp(self, the_test, TSHD_RADIUS=47):
-        if min_radius(the_test.interpolated_points) < TSHD_RADIUS:
+        if TSHD_RADIUS > min_radius(the_test.interpolated_points) > 0.0:
```

Comment:
Just to be sure. Are there cases where the road is "almost" straight but not perfectly straight (0.0)? Should we use a threshold on min/max instead of using 0.0?

Reply:
In these cases, the min radius value would be very high but not inf